### ISSUE TITLE
Update test to use toEqual instead of toBe

### ISSUE THREAD

**Issue by wesbubeck** [wesbubeck/charity-be#14]
Need to update the test to use toEqual instead of toBe to handle the values that mongoose returns from queries

**Comment by wesbubeck** [comment]
@D-Molloy  Great work sorting out a solution for the race condition with the test. I was able to find an article about how to solve this if we want to run the test in parallel. Issue #21 Is were I linked the articles. The only gain is that as the test suite grows it will speed up the test. Let discuss this and decide the priority of this. My gut says it is a nice to have and should be done down the road.

**Comment by wesbubeck** [comment]
@D-Molloy Our PRs will have some merge conflicts in the package.json file. After one goes in lets pair on the rebase to handle the merge conflicts
